torch/_inductor/compile_fx.py

```diff
@@ -771,12 +778,13 @@ def compile_fx_aot(
         if config_patches is None
         else {**config_patches, "cpp_wrapper": True}
```

Comment:
You can set `config.freezing` to True here.

Reply:
That would turn it always on. Just confirming that's what we want

Reply:
Yes. Later when when we have the weight as a separate file solution ready, we can drop it.

Reply:
okay, then we don't need the separate test if it's always enabled

Reply:
Right ?

Reply:
https://github.com/pytorch/pytorch/pull/105496/files , I have added a test_aot_inductor.py. It will be nice if you can add a test there to test a model with weights. You will need to change how we pass in args to compiled model there.